package.yaml

```diff
@@ -74,6 +80,8 @@ default-extensions:
   - ConstraintKinds
   - EmptyDataDeriving
   - TemplateHaskell
+  - NamedFieldPuns
+  - TupleSections
```

Comment:
Should these be in the defaults?

I like both of these (and have used them in places), and would be fine for adding it, but we should probably decide.

Reply:
They are there just to make my life easier and not write too much. Regardless, I think they relatively are harmless.

Reply:
Removed them all.